_sass/_layout.scss

```diff
@@ -14,11 +14,32 @@ a {
   color: #39BBD5;
 }
 
-#main-blog p {
-  text-align: justify;
-}
-#main-blog h3 {
-  font-size: 1.5em;
-  text-transform: inherit;
-  font-weight: bold;
+#main-blog{
+    
+    h3 {
+        font-size: 1.5em;
+        text-transform: inherit;
+        font-weight: bold;
+        margin-top:2rem;
+    }
+
+    p {
+        font-size: 18px;
```

Comment:
All this properties should be set in the `body` selector so that paragraphs and other regular texts use them by default.

Reply:
I don't want to change @mllocs desing, so for that I use propiety only in blog side

Reply:
Ups. Then we have a problem with the consistency of the design. Users will find a bit odd to see two different typography settings on the same website. We either stick to one design or the other but not both.